meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -273,21 +273,29 @@ PREFERRED_PROVIDER_u-boot_mx8mq ?= "u-boot-fio"
 PREFERRED_PROVIDER_u-boot-fw-utils_mx8mq = "libubootenv"
 PREFERRED_RPROVIDER_u-boot-fw-utils_mx8mq = "libubootenv"
 UBOOT_DTB_LOADADDRESS_mx8mq = "0x43000000"
-IMAGE_BOOT_FILES_mx8mq = "imx-boot ${KERNEL_IMAGETYPE} ${@make_dtb_boot_files(d)} boot.scr uEnv.txt"
-IMAGE_BOOT_FILES_sota_mx8mq = "imx-boot boot.itb"
+PREFERRED_PROVIDER_virtual/trusted-firmware-a_mx8mm ?= "imx-atf"
```

Comment:
Shouldn't this be for mx8mq not mx8mm?
```
PREFERRED_PROVIDER_virtual/trusted-firmware-a_mx8mq ?= "imx-atf"
```

Reply:
Also title has a typo:
`bsp: lmp-machine-custom: mx8qm: changes for imx-boot and uboot-fitmage class`
should be:
`bsp: lmp-machine-custom: mx8mq: changes for imx-boot and uboot-fitmage class`
(change mx8qm to mx8mq)

Reply:
nice catch, fixed!